refinery/core/search_indexes.py

```diff
@@ -65,15 +65,18 @@ def prepare_submitter(self, object):
         submitters = []
 
         for contact in investigation.contact_set.all():
-            submitters.append(contact.last_name + ", " + contact.first_name)
+            submitters.append(
+                "{}, {}".format(contact.last_name, contact.first_name)
```

Comment:
The coding style guide only says to replace the old `%`-based syntax with the new `format`-based syntax. It doesn't actually say anything about replacing a simple string concatenation with `format`. In this case it comes down to readability vs separation of style from data. I'll let you decide.